configure.py

```diff
@@ -105,10 +105,16 @@
     action="store_true",
     help="print verbose output",
 )
+parser.add_argument(
+    "--non-matching",
+    dest="non_matching",
+    action="store_true",
+    help="builds equivalent (but non-matching) or modded objects",
+)
 args = parser.parse_args()
 
 config = ProjectConfig()
-config.version = args.version
+config.version = str(args.version)
```

Comment:
What’s being passed in that’s not a str?

Reply:
`args.version` is typed as `Any` since parsed arguments are not given types ahead of time, so that's needed to convince type checking that it will always be a `str` instead of an `Optional[str]`. Without this change, the following type checking errors result:

![image](https://github.com/encounter/dtk-template/assets/29052821/e8d3fa85-69e2-4501-ae80-270f588d5d86)
![image](https://github.com/encounter/dtk-template/assets/29052821/a5e92b71-3d9a-4602-a191-da22ae26fa80)